lib/matplotlib/tests/test_axes.py

```diff
@@ -6614,6 +6614,30 @@ def test_zoom_inset():
         axin1.get_position().get_points(), xx, rtol=1e-4)
 
 
+def test_inset_polar():
+    from matplotlib.projections.polar import PolarAxes
```

Comment:
I imported the class within the test as I noticed another test in this module does that.  If it's not necessary, then I think we could parametrize these three tests into one.

Reply:
I do not see any reason to do this, I think pulling all of the imports to the top is OK.

Reply:
I moved the imports to the top, but in the end couldn't convince myself it was worth parametrizing the two small tests that are still the same: since they're so small there isn't _much_ repetition going on and I suspect readability is better as they are than with a parametrized version.